src/main/java/edu/kis/powp/jobs2d/command/manager/parsers/JSONCommandParser.java

```diff
@@ -0,0 +1,58 @@
+package edu.kis.powp.jobs2d.command.manager.parsers;
+
+import com.google.gson.Gson;
+import edu.kis.powp.jobs2d.command.DriverCommand;
+import edu.kis.powp.jobs2d.command.OperateToCommand;
+import edu.kis.powp.jobs2d.command.SetPositionCommand;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class JSONCommandParser implements Parser {
+    private static class DriverCommandAdapterJSON {
+        String name;
+        static class Command {
+            String operation;
+            int posX;
+            int posY;
+        }
+        List<Command> commands;
+    }
```

Comment:
It is ok, but just a thought. This class does not seem to be tight strongly with json. Maybe it could be reused in other (potential future) parsers?